packages/svelte/test/hydration/samples/html-tag-hydration/_before.html

```diff
@@ -0,0 +1,3 @@
+&nbsp;1
```

Comment:
not sure why this was there, let's see if it works without it
```suggestion
1
```